pkg/logflags/logflags.go

```diff
@@ -0,0 +1,25 @@
+package logflags
+
+import "strings"
+
+var Debugger = false
```

Comment:
nit: Would prefer not to export these and have package level getter methods and only mutate them during setup.

Reply:
Like this?